internal/ingress/controller/config/config.go

```diff
@@ -131,6 +131,10 @@ type Configuration struct {
 	// By default this is disabled
 	EnableAccessLogForDefaultBackend bool `json:"enable-access-log-for-default-backend"`
 
+	// EnableAuthAccessLog enable auth access log for default backend
```

Comment:
This is a global flag for all ingress objects, right? Not for the default backend

```suggestion
	// EnableAuthAccessLog enable auth access log
```